src/hooks/useIsSilenced/useIsSilenced.tsx

```diff
@@ -2,41 +2,44 @@ import { useEffect } from 'react';
 import { ParticipantIdentity } from '../../utils/participantIdentity';
 import { PARTICIPANT_TYPES } from '../../utils/rbac/ParticipantTypes';
 import useParticipants from '../useParticipants/useParticipants';
-import useParticipant from '../useParticipant/useParticipant';
 import useVideoContext from '../useVideoContext/useVideoContext';
 import { useAppState } from '../useAppState/useAppState';
+import useLocalAudioToggle from '../useLocalAudioToggle/useLocalAudioToggle';
 
 const useIsSilenced = () => {
   const {
     room: { localParticipant },
   } = useVideoContext();
   const { isSilenced, setIsSilenced, setNotification } = useAppState();
+  const [isAudioEnabled, toggleAudioEnabled] = useLocalAudioToggle();
+
   const participants = useParticipants();
-  const participantCommands = useParticipant();
 
   useEffect(() => {
     const isReporter = ParticipantIdentity.Parse(localParticipant.identity)['partyType'] === PARTICIPANT_TYPES.REPORTER;
+    if (!isReporter) return;
```

Comment:
I would add Logger.Info and Logger.End here.

Reply:
This is a hook, no need for logger